tests/test_classifier/test_rocauc.py

```diff
@@ -167,7 +167,7 @@ def test_rocauc_quickmethod(self):
         data = load_breast_cancer()
         model = DecisionTreeClassifier()
 
-        # TODO: impage comparison of the quick method 
+        # TODO: impage comparison of the quick method
```

Comment:
![imp-rage-b-color](https://user-images.githubusercontent.com/745966/27090383-0e43def2-502b-11e7-81e9-70a5702cfcbe.jpg)

I read this "imp rage" -- Google happily supplied me with this.